python/test/t_LinearModelTest_std.expout

```diff
@@ -1,7 +1,7 @@
-PartialRegressionX0Y= [class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=6.80062e-05 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=0.0952637 description=[]]
-FullRegressionX0Y= [class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=6.80062e-05 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=0.0952637 description=[]]
-PartialRegressionXY= [class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=0.00448478 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=true p-value threshold=0.1 p-value=0.756546 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=true p-value threshold=0.1 p-value=0.634249 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=0.0117222 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=true p-value threshold=0.1 p-value=0.138249 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=true p-value threshold=0.1 p-value=0.422638 description=[]]
-FullRegressionXZ= [class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=4.75917e-14 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=1.79797e-08 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=5.62147e-07 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=8.59142e-06 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=1.82078e-06 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=1.33973e-07 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=0.00271266 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=true p-value threshold=0.1 p-value=0.690999 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=true p-value threshold=0.1 p-value=0.258069 description=[],class=TestResult name=Unnamed type=Regression binaryQualityMeasure=false p-value threshold=0.1 p-value=8.07055e-15 description=[]]#10
+LinearModelFisher pvalue=0.00
+LinearModelResidualMean pvalue=1.00
+Durbin Watson =  class=TestResult name=Unnamed type=DurbinWatson binaryQualityMeasure=false p-value threshold=0.05 p-value=0.000219316 statistic=3.69566 description=[H0: auto.cor=0]
```

Comment:
It might be nice to have a test for Durbin Watson with the cases H0:auto.cor > 0 and H0:auto.cor < 0.